people/ben-cerveny/index.md

```diff
@@ -0,0 +1,13 @@
+---
+role: director
+---
+
+![ben cerveny](nebBio.jpg)
```

Comment:
Is this a good filename? What do you want this file to be named when people download it to their computer?